heltour/tournament/models.py

```diff
@@ -827,7 +829,10 @@ def is_available_for(self, round_):
     def rating_for(self, league):
         if league:
             if self.profile is None:
-                return None
+                # some admin screens cannot handle a None rating, so we return 0 instead.
+                # self.profile is only None if the player profile has never been downloaded
+                # from lichess or the account had already been closed at that first download.
+                return 0
```

Comment:
Shoudl we should return `1500`?. The logic is: ff we don't know the rating, the assumption is the default rating for lichess with a high RD, not 0, which is a completely invalid rating.

However, that means that in these views, we can't distinguish between something without a rating and and something with a legit default rating.

Reply:
I like 0 for the reason that it is an invalid rating, not only is 1500 the default rating it’s also a legitimate rating in general. It makes it very obvious that these users are a special case (and in the case of sort would be at the bottom). Plus for some of these users we already know that their last known rating was an established one, we’ve just since had the rating blanked out

Reply:
I also prefer to return a completely invalid rating (i.e. 0) that still fulfills the technical expectation of the callers of the function, i.e. to receive a numeric value:
This value should only be returned for players who closed their account while we had the bug we are fixing in  Player.update_profile. If the rest of this fix works, this line is really only to prevent admin screens for older seasons from crashing, where players are listed who closed their account in the last 1.5 years or so. Nevertheless, even if those cases show up in the future, at least it is easy to see that something is wrong with the rating.

Reply:
Works for me.